CHANGELOG.md

```diff
@@ -5,6 +5,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.4]
+
+### Added
+
+- Added the support for recursively scanning workspace directories so the
+  extension can manage multiple projects at once (#1843, #1844, #1845, #1849,
+  #1850, #1851, #1852, #1853, #1854, #1855, #1944)
+- Added a publish button to the editor view when an entrypoint is the active
+  file to allow for easier deployment of the entrypoint (#1846, #1848, #2003)
+- Added progress bars for asynchronous actions in each of the extension's views
+  (#1958)
+- Retained deployment errors in pre-Deployment files (#2004)
```

Comment:
This was already the current behavior before 1.1.4.

Reply:
Awesome I'll remove this line.

Reply:
Removed in https://github.com/posit-dev/publisher/pull/2015/commits/941fa534298165b6926c693f2aef76c04a0b85db